src/Components/SmartComponents/Modals/ReportConfigModal.test.js

```diff
@@ -25,8 +25,14 @@ const customMiddleWare = store => next => action => {
     next(action);
 }
 
+jest.mock('../../../Helpers/Hooks', () => ({
+    ...jest.requireActual('../../../Helpers/Hooks'),
+    useHybridSystemFilterFlag: () => false 
```

Comment:
I am planning to write meaningful tests for changes in ReportConficModal when I am working on RHIF-302.